test/runtests.jl

```diff
@@ -64,9 +72,9 @@ cd(dirname(@__FILE__)) do
         error("Some tests exited with errors.")
     end
 
```

Comment:
also 066c2459023fe7658494116c9eb38cde218c3348 - I think the windows build was on widely diverging branches with some infrequent merge commits when you go that early in the history.